discord/sonica-discord.py

```diff
@@ -6,26 +6,113 @@
 from discord.ext import commands
 
 from sonica_pb2_grpc import SonicaStub
-from sonica_pb2 import Empty
+from sonica_pb2 import Empty, Search, Status
 
 bot = commands.Bot(command_prefix='')
-channel = grpc.insecure_channel('localhost:7700')
-print("Trying to connect to Sonica daemon...")
-# TODO: Implement some timeout wait if the daemon isn't running or something goes wrong
-daemon = SonicaStub(channel)
+
+def surround(surround, string):
+    return f'{surround}{string}{surround}'
+
+def songformat(song):
+    return surround('**', song.title) + ' - ' + surround('*', song.artist)
 
 @bot.command()
 async def play(ctx):
-    res = daemon.Play(Empty())
-    await ctx.message.channel.send(f'result: {str(res.success)}')
+    r = daemon.Play(Empty())
+    if not r.success:
+        await ctx.message.channel.send(r.reason)
+
+@bot.command()
+async def stop(ctx):
+    r = daemon.Stop(Empty())
+    if not r.success:
+        await ctx.message.channel.send(r.reason)
+
+@bot.command()
+async def skip(ctx):
+    r = daemon.Skip(Empty())
+    if not r.success:
+        await ctx.message.channel.send(r.reason)
```

Comment:
Eventually adding a feedback message to these would be nice, so that users could see for sure whether the command was recognized or not.

Reply:
Fixed in 108d25c